[2024-12-16T18:47:58.539+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T18:47:57.332908+00:00 [queued]>
[2024-12-16T18:47:58.546+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T18:47:57.332908+00:00 [queued]>
[2024-12-16T18:47:58.547+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-16T18:47:58.559+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2024-12-16 18:47:57.332908+00:00
[2024-12-16T18:47:58.563+0000] {standard_task_runner.py:57} INFO - Started process 4692 to run task
[2024-12-16T18:47:58.566+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'sick_leave_etl_dag', 'extract_data', 'manual__2024-12-16T18:47:57.332908+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/sick_leave_dag.py', '--cfg-path', '/tmp/tmpjj3enm6w']
[2024-12-16T18:47:58.568+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask extract_data
[2024-12-16T18:47:58.583+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T18:47:58.615+0000] {task_command.py:415} INFO - Running <TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T18:47:57.332908+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T18:47:58.703+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sick_leave_etl_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T18:47:57.332908+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-16T18:47:57.332908+00:00'
[2024-12-16T18:47:58.805+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T18:47:58.815+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sick_leave_etl_dag, task_id=extract_data, execution_date=20241216T184757, start_date=20241216T184758, end_date=20241216T184758
[2024-12-16T18:47:58.859+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T18:47:58.908+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
